.github/workflows/dojoup.yml

```
@@ -0,0 +1,64 @@
+            target: x86_64-apple-darwin
+            arch: amd64
+          - os: macos-latest
+            platform: darwin
+            target: aarch64-apple-darwin
+            arch: arm64
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install curl on macOS
+        if: runner.platform == 'darwin'
```

actionlint (1.7.4)

property "platform" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)